.gitignore

```diff
@@ -1,71 +1,17 @@
-## Ignore Visual Studio temporary files, build results, and
-## files generated by popular Visual Studio add-ons.
-
 # User-specific files
 *.suo
 *.user
 *.sln.docstates
 
-# Assembly version information
-AssemblyVersionInfo.cs
-
 # Build results
-[Dd]ebug/
-[Dd]ebugPublic/
-[Rr]elease/
-x64/
 build/
-bld/
-[Bb]in/
-[Oo]bj/
-.ntvs_analysis.dat
-
-# Roslyn cache directories
-*.ide/
-
-# MSTest test Results
-[Tt]est[Rr]esult*/
-[Bb]uild[Ll]og.*
-
+bin/
+obj/
 # Visual Studio profiler
 *.psess
 *.vsp
 *.vspx
 
-# ReSharper is a .NET coding add-in
-_ReSharper*/
-*.[Rr]e[Ss]harper
-*.DotSettings.user
-
-# TeamCity is a build add-in
-_TeamCity*
-
-# DotCover is a Code Coverage Tool
-*.dotCover
-
-# NuGet Packages Directory
-NugetPackages/*
-
-# Windows Azure Build Output
-csx/
-*.build.csdef
-
-# Others
```

Comment:
Most of it was unnecessary for our existing builds. It's just some template that tries to include as many VS artifacts as possible, most of which aren't applicable to C++ projects in general.